src/interfaces/ffi/api.rs

```diff
@@ -387,7 +289,9 @@ pub unsafe extern "C" fn h_compact(
         fetch_all_entry_table_uids: Some(fetch_all_entry_table_uids),
     };
 
-    ffi_unwrap!(executor::block_on(findex.compact(
+    let rt = ffi_unwrap!(tokio::runtime::Runtime::new());
```

Comment:
for my knowledge only, was there something wrong with `executor::block_on`? now we need tokio for findex.

Reply:
1. `executor::block_on` doesn't allow multi core parallelism when doing requests. I didn't test now, but with `executor::block_on` Java wasn't doing the `fetch_chains` in parallel (massive lost in perfs).
2. `reqwest` require a "real" executor so I switch not only for FindexCloud but because of 1 I switch also the basic Findex to an executor too